app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventDetails/ui/EventDetailsViewModel.kt

```diff
@@ -243,10 +245,19 @@ class EventDetailsViewModel(
         EventDetailIdlingResourceSingleton.decrement()
     }
 
-    fun onDateClick() {
+    fun getSelectableDates(eventDate: EventDate): SelectableDates {
+        return if (eventDate.allowFutureDates) {
+            SelectableDates("12111924", "12112124")
```

Comment:
Maybe the date strings could be a constant value or default value, rather than set as plain text in different places in the application?